src/setup.py

```diff
@@ -25,15 +25,19 @@
 # 6 - ask the user delete the folder and exit
 
 
-# 1 
+# # 1 
 documents = os.environ['USERPROFILE'] + "\\Documents\\Paradox Interactive\\Hearts of Iron IV"
 try:
     os.listdir(documents)
 except Exception as e:
     print(e)
     # 1.2
-    print("Can't find the game path, please enter the path manually: ")
-    documents = input() # TODO: Check if the path is valid
+    CheckDocPathExists = False
+    while not CheckDocPathExists: 
+        documents = input("Can't find the game path, please enter the path manually: ")
+        if os.path.exists(documents): 
+            CheckDocPathExists = True
```

Comment:
You can do only one verification for `CheckDocPathExists` like here:

![image](https://github.com/ThiaudioTT/hoi4-presence/assets/43919529/e775970a-c43f-47de-a078-e96efb85a9be)
<hr>

I think the other scripts uses other directories in this folder. I will see here if we need additional verifications.

Reply:
![image](https://github.com/ThiaudioTT/hoi4-presence/assets/43919529/c3327bb5-080b-4d9b-8465-1d7c524be1be)
Yep, we need to check if a file named `settings.txt` exists in that folder.

Check that as well.

Also update the input, there's a mistake in there. It isn't the "Game path" is the documents where the game storage some files in it, like the save games